src/elements/content-uploader/ItemRemove.js

```diff
@@ -4,7 +4,8 @@
  */
 
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl } from 'react-intl';
+import type { InjectIntlProvidedProps } from 'react-intl';
```

Comment:
are we able to move this `import type` at the bottom of this list just before `import type { UploadItem, UploadStatus } [...]`?

to keep the type imports grouped together

Reply:
ESLint didn't like it... would this be something i can modify in the eslint config or is it worth ignoring the eslint warning?

<img width="800" alt="Screenshot 2022-12-19 at 2 53 01 PM" src="https://user-images.githubusercontent.com/37150601/208508782-6de9d440-772f-430d-ad1c-f5ebe0873b37.png">

Reply:
eh it's probably fine how it is